wren-ai-service/README.md

```diff
@@ -1,49 +1,104 @@
+- All the test cases are located in the `tests` folder and the following command will run all the test cases:
+  ```shell
+  make test
+  ```
+
+## Pipeline Evaluation
+
+- Copy the `config.properties.example` to `config.properties` in the `src/eval/wren-engine/etc` folder 
+  and fill in the environment variables
+- Execute the following commands to run the evaluation and specify the pipeline name as the argument:
+  ```shell
+  make eval pipeline=[pipeline-name]
+  ```
+  - pipeline-name: `ask`, `ask_detail`
+  - the evaluation results will be saved in the output folder
+- Visualize the evaluation results by executing the following command:
+  ```shell
+  make streamlit pipeline=[pipeline-name]
+  ```
+  - pipeline-name: `ask`, `ask_detail`
+- To run other ask-related evaluation: 
+  ```shell
+  poetry run python -m src.pipelines.ask.[pipeline_name]
+  ``` 
+  - e.g. `poetry run python -m src.pipelines.ask.retrieval_pipeline`
+
+## Production Environment Setup(might be deprecated in the future)
+
+- Copy `.env.prod.example` file to `.env.prod` and fill in the environment variables
 - `make build` to build the docker image
 - `make up` to run the docker container
 - `make down` to stop the docker container
 
-## Pipeline Evaluation(for development)
-
-- install `psql`
-- fill in environment variables: `.env.dev` in the src folder and `config.properties` in the src/eval/wren-engine/etc folder
-- start the docker service
-- run qdrant and wren-engine docker containers: `make run-all`
-- evaluation: `make eval-ask` and check out the outputs folder
-- `make streamlit` to compare between the evaluation results
-- to run individual pipeline: `poetry run python -m src.pipelines.ask.[pipeline_name]` (e.g. `poetry run python -m src.pipelines.ask.retrieval_pipeline`)
-
-## Demo
-
-- prerequisites
-  - install and run the docker service, and you should stop all WrenAI services first before running the demo
-  - go to the `../docker` folder and prepare the `.env.local` file
-  - make sure the node version is v16.19.0
-  - if you are using Python 3.12+, please also install `setuptools` in order to successfully install the dependencies of the wren-ui service
-- go to the `demo` folder and run `poetry install` to install the dependencies
-- in the `demo` folder, open three terminals
-  - in the first terminal, run `make prepare` to start the docker containers and `make run` to start the demo service
-  - in the second terminal, run `make ui` to start the wren-ui service
-  - in the third terminal, run `make ai` to start the wren-ai service
-- ports of the services:
-  - wren-engine: ports should be 8080
-  - wren-ai-service: port should be 5556
-  - wren-ui: port should be 3000
-  - qdrant: ports should be 6333, 6334
+## Interative the system with Demo application or UI
+
+- Go to the `../docker` folder and prepare the `.env.local` file
+- Execute the follow command to start the containers and choose App or UI to interact with the system
```

Comment:
maybe we need to list `make start-ui` and `make start-app` commands explicitly here

Reply:
also in order to make `make start-ui` work, we need to explicitly tell users these prerequisites

  - make sure the node version is v16.19.0
  - if you are using Python 3.12+, please also install `setuptools` in order to successfully install the dependencies of the wren-ui service

Reply:
in order to make make start-app to work, users should run poetry install in the demo folder first